toolchain/internal/repo.bzl

```
@@ -271,6 +271,14 @@ llvm_config_attrs.update({
         default = False,
         doc = "Use absolute paths in the toolchain. Avoids sandbox overhead.",
     ),
+    "extra_exec_compatible_with": attr.string_dict(
+        mandatory = False,
+        doc = "Extra constraints to be added to exec_compatible_with for each target, with json encoded values to get a Dict[string, List[string]]",
```

```suggestion
        doc = "Extra constraints to be added to exec_compatible_with for each target, with json encoded values to get a dict[str, list[str]]",
```
following https://docs.google.com/document/d/1Sid7EAbBd_w_T7D94Li_f_bK3zMTztFbzIMvcpzo1wY/edit